UnitsNet/UnitDefinitions/SpecificVolume.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+    "Name": "SpecificVolume",
+    "BaseUnit": "CubicMeterPerKilogram",
+    "XmlDoc": "In thermodynamics, the specific volume of a substance is the ratio of the substance's volume to its mass. It is the reciprocal of density and an intrinsic property of matter as well.",
+    "Units": [
+        {
+            "SingularName": "CubicMeterPerKilogram",
+            "PluralName": "CubicMetersPerKilogram",
+            "FromUnitToBaseFunc": "x",
+            "FromBaseToUnitFunc": "x",
+            "Prefixes": [ ],
```

Comment:
Any other units that is worth adding while at it for this quantity? I assume the operator overloads is what you are primarily using this for.

Reply:
`CubicMetersPerKilogram` is enough for us for now but if there are other units you would like to see, I can happily implement.

Reply:
Good enough for me.